pipeline_manager.py

```diff
@@ -0,0 +1,40 @@
+""" Manages logic for moving the message through the pipeline. """
+import importlib, logging
+from datetime import datetime
+from message import PipelineHistory, NotificationsMessage
+# Imports for all the pipeline steps
+from pipeline_steps.user_targeting import PipelineUserTargeting
+from pipeline_steps.user_policy import PipelineUserPolicy
+from pipeline_steps.presentation import PipelinePresentation
+from pipeline_steps.delivery import PipelineDelivery
+
+def import_from_string(class_name):
+    # TODO: Remove during refactoring
```

Comment:
This seems like either an unnecessary or an insufficient comment.  It's unnecessary because once refactoring makes it obsolete, it will be obvious that it should be removed.  It's insufficient because it doesn't say what refactoring needs to be done.  So it's neither a useful reminder for cleanup nor a useful guide to future work.